### PR TITLE
docs: updated typography

### LINKED ISSUE #1856: Update the "Typografi" page
Typografi-siden er utdatert etter endringer som ble gjort fra fluid til statiske størrelser.

### PR DESCRIPTION
**Pull request by mrosvik**
Resolves #1856

### CODE REVIEW

**Comment by mrosvik** [comment]
Feedback : Header on yellow card should be larger.

**Comment by mrosvik** [comment]
Need approving review from @mimarz or @Barsnes 😊

File: apps/storefront/pages/grunnleggende/designelementer/dynamisk-typografi.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    data={{
+      title: 'Dynamisk typografi',
+      date: '28. april 2024',
+      icon: <TokenIcon />,
+      color: 'blue',
+    }}
+  />
+);
+
+Ved hjelp av “Fluid typography” kan vi skalere tekststørrelsen gradvis mellom en min og maks-bredde, i stedet for at det blir en brå oppskalering på et spesifikt breakpoint. Den lineære funksjonen tilnærmer seg ønsket størrelse jevnt over skjermens bredde.
```
Comment (mimarz) [inline apps/storefront/pages/grunnleggende/designelementer/dynamisk-typografi.mdx:24]: brekkpunkt?

```suggestion
Ved hjelp av “Fluid typography” kan vi skalere tekststørrelsen gradvis mellom en min og maks-bredde, i stedet for at det blir en brå oppskalering på et spesifikt brekkpunkt. Den lineære funksjonen tilnærmer seg ønsket størrelse jevnt over skjermens bredde.
```

File: apps/storefront/pages/grunnleggende/designelementer/dynamisk-typografi.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- I Figma kan denne størrelsesendringen kun skje ved bruk av pluginen "Token Studio", når du har aktivert ønsket viewport for din frame.
+- Det er ikke gitt at et produkt vil bygge sitt produkt ved hjelp av dynamisk typografi. Dersom noen legger til moduler fra designsystemet i eget grensesnitt vil det ikke samsvare med skaleringen deres.
+
+<br />
+
+**Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.**
+
+## Du kan fortsatt velge å bruke fluid
+
+Du kan fortsatt velge å slå på dynamisk typografi, ved å aktivere settet "[fluid](https://github.com/digdir/designsystemet/blob/main/design-tokens/Base/Fluid.json)".
```
Comment (mimarz) [inline apps/storefront/pages/grunnleggende/designelementer/dynamisk-typografi.mdx:52]: De må vel og definere riktig viewport på framen de jobber i for at det skal fungere? De settene har vi vel kanskje fjernet helt 🤔

**Comment by Barsnes** [comment]
Should we add this to the "Siste nytt.." section on the frontpage?
We can remove "Komposisjon med..."
image

File: apps/storefront/pages/bloggen/2024/fluid-typography.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Ulemper
+
+- Halve piksler. Figma runder nedover. Noen nettlesere runder oppover, andre nedover. Det gjør at ting ikke ser likt ut i Figma og i nettleseren.
+- Du kan føle du har mindre kontroll.
+- I Figma kan denne størrelsesendringen kun skje ved bruk av pluginen "Token Studio", når du har aktivert ønsket viewport for din frame.
+- Det er ikke gitt at et produkt vil bygge sitt produkt ved hjelp av dynamisk typografi. Dersom noen legger til moduler fra designsystemet i eget grensesnitt vil det ikke samsvare med skaleringen deres.
+
+<br />
+
+**Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.**
```
Comment (Barsnes) [inline apps/storefront/pages/bloggen/2024/fluid-typography.mdx:43]:
```suggestion
Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.
```
I think we should remove bold here, or update our css to target `strong a`, so we get the correct styling for links. It is currently different:
image

**Review by mimarz — APPROVED** [review]

**Pull request merged**